internal/blueprint/customizations.go

```diff
@@ -72,8 +72,30 @@ type ServicesCustomization struct {
 }
 
 type FilesystemCustomization struct {
-	Mountpoint string `json:"mountpoint,omitempty" toml:"mountpoint,omitempty"`
-	MinSize    uint64 `json:"minsize,omitempty" toml:"size,omitempty"`
+	Mountpoint string `json:"mountpoint,omitempty"`
+	MinSize    uint64 `json:"minsize,omitempty"`
+}
+
+func (fsc *FilesystemCustomization) UnmarshalTOML(data interface{}) error {
```

Comment:
We also support JSON so we need a JSON unmarshaller.

Reply:
This is mentioned in the commit message. If this feature is mainly for the interactive usage, it doesn't need to be implemented for JSON. But I'm fine implementing the same for JSON.